initial_data_load/scripts/export-mongo-collections.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+#
+# Simple script to export the sic_code collections
+#
+
+set -o errexit  # abort on nonzero exitstatus
+set -o pipefail # don't hide errors within pipes
+
+if [[ -z "${SIC_CODE_API_DATABASE}" ]]; then
+    echo "ERROR: SIC_CODE_API_DATABASE environmental variable NOT set"
+    exit 1
+fi
+
+if [[ -z "${SIC_CODE_API_MONGO_URL}" ]]; then
+    echo "ERROR: SIC_CODE_API_MONGO_URL environmental variable NOT set"
+    exit 1
+fi
+
+script_dir=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+import_files=${script_dir}/../import_files
+echo "Writing import files to ${import_files}"
+
+export_collection() {
+    local mongo_collection="$1"
```

Comment:
I think that you can skip the quote in direct assignment like this:
local mongo_collection=$1

[Unix Stack exchange: When is double quoting necessary?](https://unix.stackexchange.com/a/68748)

Reply:
Same point applies elsewhere.

Reply:
I won't go overboard with prining these double quotes since the reference says

it is far easier to use double quotes all the time than to remember when they are needed. They are needed most of the time, so you'll need to learn when they aren't needed, not when they are needed.

Reply:
Agreed. Then go the whole hog and use braces too.